datahub-web-react/src/app/entity/view/ManageViews.tsx

```diff
@@ -20,7 +20,9 @@ const PageTitle = styled(Typography.Title)`
     }
 `;
 
-const ListContainer = styled.div``;
+const ListContainer = styled.div`
+    height: calc(100% - 120px);
```

Comment:
Can you explain where 120px originates?

Reply:
can be replace with height: calc(100% - 11vh); after calculating PageHeaderContainer and Pagination